webapp/src/components/conference/conference.tsx

```diff
@@ -163,6 +164,7 @@ export default class Conference extends React.PureComponent<Props, State> {
         this.api.executeCommand('hangup');
         setTimeout(() => {
             this.props.actions.openJitsiMeeting(null, null);
+            this.props.actions.setUserStatus(this.props.currentUserId, 'online');
```

Comment:
Is there a constant that can be used here?

Reply:
I actually wanted to use an enum for this. Does that make sense ?

Reply:
@maisnamrajusingh Can you use the one define in mattermost-redux: https://github.com/mattermost/mattermost-redux/blob/40368bb807b8b5a3dc0f4ce4a5a76eabc5c9de54/src/constants/general.ts#L21-L21

Reply:
@maisnamrajusingh please see above